src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -48,11 +52,11 @@ For more about these tools, see the [new user management docs](/docs/accounts/ac
 
 The migration wizard is meant to have all the context you need to complete the migration process but here are some tips for potential areas of confusion:
```

Comment:
maybe "The migration wizard provides all the context you need to complete the migration process, but here are some additional tips."